### DISCUSSION TITLE
How to use TWO connections?

### DISCUSSION THREAD

**Discussion by NicoMachado** [strapi/strapi#7215] [Legacy Topics]
I am using strapi to get data as a RESTfull API, and it looks great indeed!

But, the database where I get data is from another app, so I can't change their structure.

And Strapi add some tables for the user/permision modules.

I created manually another entry in : config\database.js

## This is default Entry
```
default: {
  connector: 'bookshelf',
  settings: {
    client: 'mysql',
    host: env('DATABASE_HOST', '127.0.0.1'),
    port: env.int('DATABASE_PORT', 3306),
    database: env('DATABASE_NAME', 'dblogic'),
    username: env('DATABASE_USERNAME', 'root'),
    password: env('DATABASE_PASSWORD', 'not_empty'),
    ssl: env.bool('DATABASE_SSL', false),
  },
  options: {}
},
```
## This is My new Entry
```
atdbconnection: {
  connector: 'bookshelf',
  settings: {
    client: 'mysql',
    host: env('DATABASE_HOST', '127.0.0.1'),
    port: env.int('DATABASE_PORT', 3306),
    database: env('DATABASE_NAME', 'dblogic'),
    username: env('DATABASE_USERNAME', 'root'),
    password: env('DATABASE_PASSWORD', 'not_empty'),
    ssl: env.bool('DATABASE_SSL', false),
  },
  options: {}
},
```

In another post somebody said that should be a database selector, for each Content Type entry, but I can't see anyone.

It's posible to use 2 connections in Strapi ?

Any help will be appreciated !!

Best Regards

**Comment by lauriejim** [comment]
Hello! The 2 databases should be the same type (2 mysql or 2 postures for example)
And this will work only with SQL database.
Then in your model settings file, you will have to update the `connection` key.
Did you try it?

**Reply by NicoMachado** [reply]
Hi, @lauriejim ..
yes, my idea is to keep the same mySql Server, but different ddbb.
 i'll try it and let you know!

**Reply by NicoMachado** [reply]
Yes! It works.
Where I can find information on this two field created: created_at, updated_at ?
And differences between dev mode and production mode..
Best Regards

**Reply by lauriejim** [reply]
For now, we don't have docs about that :/
I have to work with the dev team to add more content about that in the docs.

**Reply by lauriejim** [reply]
And I don't know enough myself (for now) to help on that part.
